app/enums/DocumentType.scala

```diff
@@ -28,6 +28,14 @@ case object Poa2Charge extends DocumentType {
   override val key: String = "ITSA - POA 2"
 }
 
+case object Poa1Reconciliation extends DocumentType {
```

Comment:
If we can add "Debit" as a suffix to the type name, as this would improve readability etc